pkg/controller/buildrun/generate_taskrun_test.go

```diff
@@ -55,10 +55,10 @@ var _ = Describe("GenerateTaskrun", func() {
 				build, err = ctl.LoadBuildYAML([]byte(test.MinimalBuildahBuild))
 				Expect(err).To(BeNil())
 
-				buildRun, err = ctl.LoadBuildRunYAML([]byte(test.MinimalBuildahBuildRun))
+				buildRun, err = ctl.LoadBRFromBytes([]byte(test.MinimalBuildahBuildRun))
```

Comment:
Just for consistency, I would like to see this being called something like `LoadBuildRunFromBytes`.

Reply:
weird, but that change is already there. I dont recall how